Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1235,11 +1235,10 @@ class BrowserViewController: UIViewController {
     }
 
     private func getCurrentUIState() -> UIState {
-        guard let tab = tabManager.selectedTab,
-        let displayURL = tab.displayURL where displayURL.absoluteString.characters.count > 0 else {
-            if let homePanelController = homePanelController {
-                return .HomePanels(homePanelState: homePanelController.homePanelState)
-            }
+        if let homePanelController = homePanelController {
+            return .HomePanels(homePanelState: homePanelController.homePanelState)
+        }
+        guard let tab = tabManager.selectedTab where !tab.loading else {
```

Comment:
I am kinda concerned about the removal of the displayURL check due to this comment in the original commit of the displyURL check

``
Better detect whether or not we are displaying home panels by checking the displayURL of the  selected tab rather than whether or not there is a home panel controller. this is because a timing issue sometimes meant that the home panel controller from a previous tab was still hanging around when requesting a menu from a new tab with a URL, therefore displaying the wrong URL
```
trying to find the STR for this to see if it's still a problem....